text/0000-road-to-10.md

```diff
@@ -57,7 +57,11 @@ There are a number of issues that are relatively small on their own, but combine
 
 ## Yarn Plug'n'Play
 
-Another feature that could be considered for 1.0 is support for Yarn Plug'n'Play, which installs all of the dependencies from a local cache and doesn't use `node_modules`. However, we haven't yet received requests to support that functionality, and it will require a fair amount of investigation on the implementation. One major issue is that Yarn's feature relies on JS Monkeypatching, which we can't take advantage of in Rust, so we would need to either call Node (which will have performance implications) or duplicate the Yarn logic within Volta (which will take significant development effort).
+A feature that could be considered for 1.0 is support for Yarn Plug'n'Play, which installs all of the dependencies from a local cache and doesn't use `node_modules`. However, we haven't yet received requests to support that functionality, and it will require a fair amount of investigation on the implementation. One major issue is that Yarn's feature relies on JS Monkeypatching, which we can't take advantage of in Rust, so we would need to either call Node (which will have performance implications) or duplicate the Yarn logic within Volta (which will take significant development effort).
```

Comment:
Note that while we haven't *yet* hit this, some puzzle pieces *should* 🤞 fall into place in the Ember community (many of our early adopters) over the next few months that unlock Yarn P'n'P there, making adoption for some of our users much likelier.

Reply:
Good to know :+1: We'll definitely need to keep an eye on that and bump it up in priority if it becomes a requested feature.